toolflow/vivado/platform/AU50/AU50.tcl

```diff
@@ -0,0 +1,205 @@
+# Copyright (c) 2014-2020 Embedded Systems and Applications, TU Darmstadt.
+#
+# This file is part of TaPaSCo
+# (see https://github.com/esa-tu-darmstadt/tapasco).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+
+namespace eval platform {
+  set platform_dirname "AU50"
+  set pcie_width "x16"
+
+  if { [::tapasco::vivado_is_newer "2020.1"] == 0 } {
```

Comment:
Is the U50 now directly available in Vivado? In branch `feature/alveo-u50` I had to use the XilinxBoardStore...

Reply:
Nope, that's part of the users Vivado installation. Not sure if adding it to TaPaSCo is a good solution to this problem. I could add an information where to get the necessary files.

Reply:
Then maybe we can include my changes to automatically download it from the XilinxBoardStore. We should avoid adding the board files to the TaPaSCo repo.

Reply:
Are you sure the AU50 is still included by default? I can only find it for very specific Vivado versions, e.g. https://github.com/Xilinx/XilinxBoardStore/tree/2020.1.1/boards/Xilinx/au50/production/1.0 :/

Reply:
You are right :-/ Back then I thought that it was this way because it is a new card. But apparently, Xilinx really does not want people to use the RTL flow...
The board files have Apache 2.0 license, which could be included in our repo.

Reply:
The board files seem to be version specific. When using them with Vivado 2020.1 it works fine, but with Vivado 2020.2 it fails at the `apply_bd_automation` statements for the PCIe core.
So adding them to the repo is probably not an option

Reply:
2020.1 was the only version I checked it with. This is version to version difference is very annoying :/

What kind of error are you seeing?

Reply:
The first `apply_bd_automation` in `create_pcie_core` fails with the following:
```
ERROR: [IP_Flow 19-3461] Value 'GTH_Quad_128' is out of the range for parameter 'select quad(select_quad)' for BD Cell 'host/axi_pcie3_0' . Valid values are - GTY_Quad_227, GTY_Quad_226, GTY_Quad_225, GTY_Quad_224
ERROR: [BD 41-245] set_property error - Value 'GTH_Quad_128' is out of the range for parameter 'select quad(select_quad)' for BD Cell 'host/axi_pcie3_0' . Valid values are - GTY_Quad_227, GTY_Quad_226, GTY_Quad_225, GTY_Quad_224
Customization errors found on 'host/axi_pcie3_0'. Restoring to previous valid configuration.
```

Reply:
Did they really change the Quad naming? What an odd thing to do.